server/norace_test.go

```diff
@@ -2000,3 +2000,154 @@ func TestNoRaceJetStreamClusterSuperClusterRIPStress(t *testing.T) {
 		time.Sleep(10 * time.Millisecond)
 	}
 }
+
+func TestJetStreamSlowFilteredInititalPendingAndFirstMsg(t *testing.T) {
```

Comment:
Since this is in the `norace_test.go` that we want to run only if not passing `-race` to `go test`, the name of the test has to start with `TestNoRace`. This is because on travis we do:
```
go test -v -run=TestNoRace --failfast -p=1 ./...
```
While at it, can you also change the name of this test: `TestJetStreamClusterMirrorExpirationAndMissingSequences` so it starts with `TestNoRace`

Reply:
ok will do.